Makefile

```diff
@@ -74,7 +74,12 @@ list-hd:
 	echo counted $$LIST_HD usages ;\
 	test $$LIST_HD -eq 300
 
-quality-gate: list-hd ;
+verify-cert:
+	@NONE=$$( git grep -r --count 'verify_cert:None' | awk 'BEGIN { FS=":" } { c += $$2 } END { print c }') ;\
+	echo "counted $$NONE usages of verify_cert:None" ;\
+	test $$NONE -eq 8
```

Comment:
```suggestion
	@NONE=$$(git grep -r --count '~verify_cert:None' -- **/*.ml | cut -d ':' -f 2 | paste -sd+ - | bc) ;\
	echo "counted $$NONE usages of verify_cert:None" ;\
	test $$NONE -eq 6
```

Reply:
using awk along the .ml filter returns '2' for some reason.

Reply:
You are right - the code needs to restrict the grep to `*.ml` files as otherwise the Makefile is counted as well - just found out.